.github/workflows/test.yaml

```diff
@@ -0,0 +1,29 @@
+name: test
+on:
+  pull_request:
+    branches: [develop]
+    paths:
+      - '**.tsx?'
+      - '**.test.ts'
+
+env:
+  NODE_ENV: dev
+  ACTIONS_ALLOW_UNSECURE_COMMANDS: ${{ secrets.ACTIONS_ALLOW_UNSECURE_COMMANDS }}
```

Comment:
I would use the word insecure

Reply:
This is github syntax, not something I made up :)